src/Uploader/Uploader.tsx

```diff
@@ -215,7 +215,7 @@ class Uploader extends React.Component<UploaderProps, UploaderState> {
 
   handleAjaxUploadError = (
     file: FileType,
-    status: object,
+    response: object,
```

Comment:
This is not a mis-naming. The `handleAjaxUploaderError` function is indeed called with an `ErrorStatus` object instead of a `response` as shown here https://github.com/rsuite/rsuite/blob/v4/src/utils/ajaxUpload.ts#L14